### PR TITLE
Metadata SEO (titles + descriptions)

### PR DESCRIPTION
**Pull request by joenyzio**
Adding titles, descriptions, and formatting updates to all documentation. 

Goals 
- To clearly define the purpose of each document and make the docs easier to read and navigate 
- Add SEO elements to frontmatter to improve the visibility of docs content
- Keep note potential improvements and identify missing information in each doc

### CODE REVIEW

**Comment by joenyzio** [comment]
Completed Welcome and tutorials section. Continuing as draft.

File: docs/celo-codebase/protocol/bridging/bridging-native-assets.md [hunk 1]
```diff
@@ -1,60 +1,63 @@
...
-
-## Step 1: Call the EthHelper
+How to bridge native assets from ETH and Matic to Celo with Etherscan.
 
-1. Navigate to the [Etherscan](https://etherscan.io/) page for the EthHelper.
+___
+## Call the EthHelper
 
-:::info
+Navigate to the [Etherscan](https://etherscan.io/) page for the EthHelper.
```
Comment (critesjosh) [inline docs/celo-codebase/protocol/bridging/bridging-native-assets.md:16]: What do you think about updating the link to direct to the appropriate smart contract for each network and moving the link into the tab for that network?

File: docs/celo-codebase/protocol/bridging/bridging-tokens-with-etherscan.md [hunk 2]
```diff
@@ -50,48 +48,56 @@ Approving too much is usually ok, but not approving enough will cause your next
...
 
-:::info
 
-* **On Ethereum:** 0x6a39909e805A3eaDd2b61fFf61147796ca6aBB47
-* **On Polygon:** 0xf244eA81F715F343040569398A4E7978De656bf6
-
-:::
+<Tabs>
+  <TabItem value="Ethereum" label="On Ethereum" default>
+    Address = 0x6a39909e805A3eaDd2b61fFf61147796ca6aBB47
```
Comment (critesjosh) [inline docs/celo-codebase/protocol/bridging/bridging-tokens-with-etherscan.md:62]: add links directly to the contracts on etherscan here as well?

File: docs/celo-codebase/protocol/bridging/bridging-native-assets.md (hunk 1 above)
Comment (critesjosh) [reply docs/celo-codebase/protocol/bridging/bridging-native-assets.md]: i made the update

**Review by critesjosh — APPROVED** [review]
lgtm!

**Pull request merged**